Source/SpacePort/SpacePort.csproj

```diff
@@ -6,6 +6,26 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Remove="ParkingComponents\**" />
+    <EmbeddedResource Remove="ParkingComponents\**" />
+    <None Remove="ParkingComponents\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="DBAssets\DbModels\NewFile1.txt" />
```

Comment:
What is this?

Reply:
Problem med LiveShare, vi kommer inte använda det framöver, det skapade väldigt mycket problem.